test/js/css-image-value.js

```diff
@@ -0,0 +1,28 @@
+suite('CSSImageValue', function() {
```

Comment:
You should test somewhere that a developer can query an instance of ImageValue using the "instanceof" operator, and get the right result with the exposed CSSImageValue constructor, e.g.

```javascript
if (a instanceof CSSImageValue)
```

or

```javascript
if (a instanceof CSSResourceValue)
```
or
```javascript
if (a instanceof CSSStyleValue)
```

There are various other tests which do similar things using the assert.instanceOf helper.